### ISSUE TITLE
Easy digestible results

### ISSUE THREAD

**Issue by joshkurz** [axemclion/browser-perf#5]
This is a nice product and I believe you are headed in the good direction, but there are some cases that I would mention, which I have not been able to find. 

Is there any plan to use this project in a way that is more along the lines of jsperf. Meaning could this perf library run tests many times to generate an aggregate result, which would output some percentage difference between linked test cases?

It seems that to be useful we need numbers to compare. Should a third party library offer a simple plug and play ability to compare results via CLI? I think so... 

A test that runs and has some various perf metrics is nice, but is not usable unless those metrics are compared against either historical application data or other tests that also run during the same testing cycle. 

so in summary I am thinking about a library that does the following:
1. Ability to run 'it' logical blocks many times to generate an aggregate output.
2. Ability to link tests together to easily compare outputs.
3. Ability to simply compare against historical local/remote performance data.
4. Ability to output simple visualizations that compare specified test outputs.
